metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "simp-rsync",
-  "version": "6.2.0",
+  "version": "6.3.0",
   "author": "SIMP Team",
   "summary": "manage and rsync server, secured by stunnel",
```

Comment:
I think you mean `manage an rsync server` rather than `manage and rsync server`, don't you?